packages/seeder/src/drive.js

```diff
@@ -124,6 +125,22 @@ class Drive extends EventEmitter {
     }
   }
 
+  async info () {
+    // returns drive info, ie: { version, index.json }
+    if (!this._ready) {
+      return {}
+    }
+    let indexJSON = {}
+    try {
+      indexJSON = JSON.parse(await this._hyperdrive.readFile('index.json', 'utf-8'))
+    } catch (_) {}
+    const version = this._hyperdrive.version
+    return {
+      version,
+      indexJSON
+    }
```

Comment:
Just a little of space :smile: 
```suggestion
    if (!this._ready) {
      return {}
    }
    
    let indexJSON = {}
    
    try {
      indexJSON = JSON.parse(await this._hyperdrive.readFile('index.json', 'utf-8'))
    } catch (_) {}
    
    const version = this._hyperdrive.version
    
    return {
      version,
      indexJSON
    }
```